.github/workflows/client.yml

```diff
@@ -33,3 +33,12 @@ jobs:
 
       - name: Build
         run: ./gradlew compileDebugAndroidTestSources check assemble ktlintCheck
+
+      - name: Upload APKs
+        uses: actions/upload-artifact@v3
```

Comment:
Is this executed for all branches/forks or only for master/main?

Reply:
It's executed for all. The links in this PR are to my fork where this ran on a non-standard branch `gh-actions`